gitgud/levels/mixedbag/_onecommit/setup.spec

```diff
@@ -0,0 +1,5 @@
+1 (master)
+2 : 1 (debug)
```

Comment:
This would be the perfect opportunity to use tags instead of commits. To specify a tag, do `(tag: debug)`. Do this for debug/printf for both spec files